src/app/plugins/register.plugin.ts

```diff
@@ -17,16 +17,34 @@ export class RegisterPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args?: string[]) {
-    const request = this.container.classService.buildRequest(message.author, args);
-    if (!request) {
-      message.reply('I was unable to build your request.');
+    if (!args) {
       return;
     }
-    try {
-      const response = await this.container.classService.register(request);
-      message.reply(response);
-    } catch (e) {
-      message.reply(e);
+
+    const successfulClasses: string[] = [];
+    const invalidClasses: string[] = [];
+    for (const arg of args) {
+      const request = this.container.classService.buildRequest(message.author, [arg]);
+      if (!request) {
+        message.reply('I was unable to build your request.');
+        return;
+      }
+      try {
+        const response = await this.container.classService.register(request);
+        if (response.includes('success')) {
+          successfulClasses.push(arg);
+        } else {
+          invalidClasses.push(arg);
+        }
+      } catch (e) {
+        message.reply(e);
+      }
+    }
+
+    let messageForUser = `Successfully added to ${successfulClasses.length} classes`;
```

Comment:
then u will need to handle the cases of when no classes were added successfully (just have specific message if len == 0)